src/render/camera.h

```diff
@@ -66,9 +66,15 @@ typedef struct camera_state_s
 
 typedef struct camera_s
 {
+#ifdef __GNUC__
```

Comment:
so much copy-paste code (original OT source problem)...
hmmm OK, I will fix `__attribute__((packed, aligned(16)))` soon
timer is more complex problem (P.S. VS has some posix like time functions; P.P.S. `QueryPerformanceFrequency` is inside SDL)

Reply:
you can close it anytime, it's not really needed, more like references for windows error at compile :)